src/server/routes/register.ts

```diff
@@ -37,7 +47,7 @@ router.get(
     const html = renderer(Routes.REGISTRATION_EMAIL_SENT, {
       requestState: deepmerge(state, {
         pageData: {
-          email: getEmailFromPlaySessionCookie(req),
+          email: getEmailFromPlaySessionCookie(req) || readGUEmailCookie(req),
```

Comment:
With this change, the email sent page will now be used for Gateway, not just the MVP0 `identity-frontend` project as it is now. So it'll also need to read from the `GU_email` cookie.

Once `identity-frontend` decommissioned we can reduce this down to just the `GU_email` cookie.